frontend/public/components/deployment-config.tsx

```diff
@@ -262,6 +264,29 @@ export const DeploymentConfigDetailsList = ({ dc }) => {
   );
 };
 
+export const DeploymentConfigDeprecationAlert: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      isInline
+      variant="info"
+      title={t('public~DeploymentConfigs is being deprecated with OpenShift 4.14')}
```

Comment:
Either:
```suggestion
      title={t('public~DeploymentConfigs are being deprecated with OpenShift 4.14')}
```

or 

```suggestion
      title={t('public~DeploymentConfig is being deprecated with OpenShift 4.14')}
```

@opayne1 thoughts?

Reply:
I would go with the second

>  title={t('public~DeploymentConfig is being deprecated with OpenShift 4.14')}